### PR TITLE
Red Hat cheatsheet: Added aliases and naming consistency

### PR DESCRIPTION
**Pull request by tagawa**
Look at the official site, they seem to be called "Red Hat" (with a space).
Also added the aliases as this cheat sheet is specifically about yum.

---
https://duck.co/ia/view/redhat_cheat_sheet
Maintainer: @ankushg07

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=redhat cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/redhat_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=redhat cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/redhat_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=redhat cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/redhat_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/redhat.json [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
+        "sourceName": "Red Hat",
         "sourceUrl": "https://access.redhat.com/articles/yum-cheat-sheet"
     },
+    "aliases": [
+        "red hat", "redhat yum", "redhat enterprise yum", "red hat yum", "red hat enterprise yum"
+    ],
     "template_type": "terminal",
     "section_order": [
-        "Yum Queries",
+        "Yum queries",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/redhat.json]: The section name below also need to be updated so they match :+1: 

Right now this causes errors

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=redhat cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/redhat_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/redhat.json (hunk 1 above)
Comment (tagawa) [inline share/goodie/cheat_sheets/json/redhat.json]: Oops, overlooked that. Thanks.

**Comment by ankushg07** [comment]
@tagawa  @moollaza Will surely rectify my mistakes 
Could you please guide me how to add alias for cheat sheets

**Comment by tagawa** [comment]
@ankushg07 It's OK! I've submitted the pull request with updates so there's nothing for you to do.
If you're interested, for future reference, the aliases are just a small extra section in the JSON file. You can see the ones I added here:
https://github.com/duckduckgo/zeroclickinfo-goodies/pull/2883/files#diff-8fbcafa21a0f5d4a8b8d7304348fa7f5R9

**Comment by GuiltyDolphin** [comment]
@ankushg07 Are you happy with this?

**Comment by ankushg07** [comment]
@GuiltyDolphin any mistakes or problem you encountered?

File: share/goodie/cheat_sheets/json/redhat.json (hunk 1 above)
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/redhat.json]: @tagawa Why did we go for lowercase? I think `Yum Queries` looks much nicer.

Comment (tagawa) [inline share/goodie/cheat_sheets/json/redhat.json]: @GuiltyDolphin To be consistent with the other section headings (Package maintenance, Options for the yum command).

Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/redhat.json]: @tagawa How about updating them all to be caps on important words?

Comment (tagawa) [inline share/goodie/cheat_sheets/json/redhat.json]: OK, have done. Seems this is pretty inconsistent across various cheat sheets.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=redhat cheat sheet)
                
                [Mobile screenshot](https://images.duckduckgo.com/iu/?u=https://ia-screenshots.s3.amazonaws.com/redhat_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/redhat.json (hunk 1 above)
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/redhat.json]: @tagawa :cry: Yeah - well we do have like... 300 ^.^

**Comment by GuiltyDolphin** [comment]
@tagawa @ankushg07 Thanks guys! :dolphin:

**Pull request merged**